core/domain/rights_manager.py

```diff
@@ -113,7 +113,11 @@ def validate(self):
                 editor_viewer)
 
     def to_dict(self):
-        """Returns a dict suitable for use by the frontend."""
+        """Returns a dict suitable for use by the frontend.
+
+        Returns:
+            ActivityRights in dict format.
```

Comment:
We typically write the type before describing the object, both in the Returns part and the Args part, so this should be
`dict. A dict version of ActivityRights suitable for use by the frontend.`
and similarly for everything else in this PR.